en/mkdocs.yml

```diff
@@ -106,6 +106,14 @@ nav:
           - Programmatic Access to Choreo Insights API: observe-and-analyze/analyze/programmatic-access-choreo-insights-api.md
   - Alerts: alerts.md
   - DevOps: devops/devops-portal.md
+  - iPaaS:
+      - Getting Started: ipaas/getting-started.md
```

Comment:
Change "- Getting Started: ipaas/getting-started-with-choreo-ipaas.md" to "- Get Started: ipaas/get-started-with-choreo-ipaas.md"